pandas/core/reshape/concat.py

```diff
@@ -352,8 +352,8 @@ def __init__(
         for obj in objs:
             if not isinstance(obj, (Series, DataFrame)):
                 msg = (
-                    "cannot concatenate object of type '{typ}'; "
-                    "only Series and DataFrame objs are valid".format(typ=type(obj))
+                    f"cannot concatenate object of type '{type(obj)}'; "
+                    f"only Series and DataFrame objs are valid"
```

Comment:
```suggestion
                    "only Series and DataFrame objs are valid"
```

Don't need the f-prefix for lines with no replacement